src/main/java/core/basesyntax/service/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, File file) {
```

Comment:
The method parameter 'File file' should be replaced with a 'String filePath' to avoid hardcoding the file resource and to comply with the checklist's recommendation on avoiding hardcoded values.